app/src/main/java/org/ole/planet/myplanet/ui/team/MyTeamsDetailFragment.kt

```diff
@@ -94,7 +97,6 @@ class MyTeamsDetailFragment : BaseNewsFragment() {
         }
         v.findViewById<View>(R.id.add_message).setOnClickListener { showAddMessage() }
     }
-
```

Comment:
Also, not sure why this line was removed, but I think it would be nice to have a line of spacing between the functions.